### PR TITLE
DIABLO-51, /admin view lists capture-enabled rooms; instructor will be routed to /course/:section_id

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/DIABLO-51

I believe Diablo the webapp will rely on only one Salesforce query: `get_capture_enable_rooms`. This PR gives admin users a default dashboard view which will evolve. I renamed `authorized_users` table because instructor UIDs will not be kept in the db and yet some instructors will be authorized users.

### CODE REVIEW

File: diablo/api/course_capture_controller.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+@app.route('/api/capture/enabled_rooms')
+def get_enabled_rooms():
+    rooms, total_room_count = get_capture_enabled_rooms()
+    return tolerant_jsonify({
+        'rooms': rooms,
+        'totalRoomCount': total_room_count,
+    })
+
+# @app.route('/api/capture/enabled_rooms')
```
Comment (paulkerschen) [inline diablo/api/course_capture_controller.py]: @johncrossman Was this temporary for testing, or is it staying in as a reminder spec?

Comment (johncrossman) [reply diablo/api/course_capture_controller.py]: Oops!  That was a mock endpoint that should have been removed. Amending

File: .travis.yml [hunk 2]
```diff
@@ -14,8 +14,11 @@ jobs:
     - stage: test
       script:
         - psql -c 'create database diablo_test;' -U postgres
+        - psql -c 'create database diablo_loch_test;' -U postgres
         - psql diablo_test -c 'create extension pg_trgm;' -U postgres
         - psql diablo_test -c 'create role diablo superuser login; alter schema public owner to diablo;' -U postgres
+        - psql diablo_loch_test -c 'create extension pg_trgm;' -U postgres
+        - psql diablo_loch_test -c 'alter schema public owner to diablo;' -U postgres
```
Comment (johncrossman) [inline .travis.yml]: Amended this PR to include data-loch queries.

File: diablo/__init__.py [hunk 3]
```diff
@@ -54,3 +64,17 @@ def std_commit(allow_test_environment=False):
...
+def cachify(key_pattern):
+    @decorator
+    def _cachify(func, *args, **kw):
+        args_dict = get_args_dict(func, *args, **kw)
+        key = key_pattern.format(**args_dict)
+        cached = cache.get(key)
+        if cached is None:
+            cached = func(*args, **kw)
+            cache.set(key, cached)
+        return cached
```
Comment (johncrossman) [inline diablo/__init__.py]: Predictable cache keys make it easier to mock in pytests.

**Pull request merged**